py/server/requirements.txt

```
@@ -1,2 +1,2 @@
-setuptools~=58.2.0
-wrapt~=1.13.1
\ No newline at end of file
+setuptools~=78.1.1
+wrapt~=1.13.1
```

This is what is in https://github.com/deephaven/deephaven-core/pull/7321. We don't even use `wrapt` anymore.  

```suggestion
pyarrow
numpy
pandas
jpy
jedi
docstring-parser
```